Formula/rdkit.rb

```diff
@@ -0,0 +1,72 @@
+class Rdkit < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://rdkit.org/"
+  url "https://github.com/rdkit/rdkit/archive/Release_2020_09_4.tar.gz"
+  sha256 "9e734ca8f99d8be1ef2ac51efb67c393c62e88b98cfa550d6173ce3eaa87b559"
+  license "BSD-3-Clause"
+  head "https://github.com/rdkit/rdkit.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "eigen"
+  depends_on "freetype"
+  depends_on "numpy"
+  depends_on "postgresql@12"
```

Comment:
Does it not work with the latest?

Reply:
Thanks, @SMillerDev. This rdkit version was already compatible with postgresql 13.